server/app/views/admin/questions/MapQuestionSettingsFilterFragment.html

```diff
@@ -0,0 +1,47 @@
+<th:block
+  th:fragment="filterInput(index, possibleKeys, selectedKey, displayNameValue)"
+>
+  <div class="display-flex flex-justify filter-input">
```

Comment:
(non-blocking) I would recommend creating a `th:with` here and make local variables for the array names.